### PR TITLE
added documentation and tests how to use requestmock as mock.patch

### LINKED ISSUE #31: Allow applying @responses.activate to a class
Right now I have to apply the decorator to every method I want to mock responses in; I'd prefer to apply the decorator only to the `TestCase` declaration as my `TestCase` does http in every test method. This would keep requests consistent with the mock.patch behaviour.

E.g.:

```
@responses.activate
class CommandHandlerTestCase(unittest.TestCase):

    def test_function_1(self):
        responses.add(...)

    def test_function_2(self):
        responses.add(...)

    def test_function_3(self):
        responses.add(...)
```

### PR DESCRIPTION
**Pull request by beliaev-maksim**
closes #31

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -732,8 +732,11 @@ Responses as a context manager
         resp = requests.get('http://twitter.com/api/1/foobar')
         resp.status_code == 404
 
-Responses as a pytest fixture
------------------------------
+Using ``responses`` in unit tests
+---------------------------------
```
Comment (markstory) [inline README.rst:736]: Is this additional heading necessary? Responses is really only used in unit tests.

Comment (beliaev-maksim) [reply README.rst:736]: @markstory 
we need some title to group. May be you can suggest something?

core idea is that this chapter describes integration with `pytest` and `unittest`

Comment (beliaev-maksim) [reply README.rst:736]: @markstory 
maybe `integration with unit test frameworks` ?

Comment (beliaev-maksim) [reply README.rst]: I did a rename

**Pull request merged**